docs/next/quickstart.mdx.skill.md

```
@@ -0,0 +1,254 @@
+## 1. Create the project
+
+```bash
+iii project init quickstart --template quickstart
+cd quickstart
+```
+
+This creates the two workers that you'll run: a Python worker that adds two numbers and stores the
+sum in state, and a TypeScript worker that exposes an http endpoint and calls the Python worker
+through the iii engine.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)